mission.md

```diff
@@ -0,0 +1,332 @@
+
+
+## Context
+
+We have always had a
+["_Why_?"](https://github.com/dwyl/start-here#why)
+goal (_or several goals depending on who you ask_) for @dwyl.
+
+_Long_ before we started @dwyl we had been working for charities
+and non-profit organisations on a variety of missions
+including poverty reduction, cancer research, homelessness,
+fair trade and education.
+
+One of the things that became clear to us
+while we were working for various charities
+and for-profit businesses
+was that in _most_ cases time was not used effectively
+and communication channels were poor.
+Even though mission statements were clear,
+_translating_ those words into the day-to-day actions
+that most _effectively_ achieved the mission
+was severely lacking in most cases.
+
+
+
+## _However ..._
+
+Even though our **`start-here`** repo has been read and "_starred_" by _many_ people:
+
+![start-here-stars-sept-2019](https://user-images.githubusercontent.com/194400/65373907-8b706d80-dc7b-11e9-8f6b-d844b9a975c8.png)
+
+(_evidence that there is **some clarity** in our **purpose**_)
+
+We still feel that we are doing a _poor_ job of _communicating_ the **`Mission`**
+to the extent that people have _asked_ us to clarify it.[<sup>2</sup>](#notes)
+
+Our objective with defining a _single_ clear **`Mission`** for the organisation is to help _everyone_ in the community/team _understand exactly_ what we are trying to achieve and how much _effort_ and _perseverance_ it will require along the way.
+
+
+
+# _What_ is a `Mission` (Statement)?
+
+
+
++ https://en.wikipedia.org/wiki/Mission_statement [6 min read]
++ https://www.entrepreneur.com/encyclopedia/mission-statement [2 min read]
+
+## Examples
+
+### Unicef
+
+"_advocate for the protection of children's rights,
+help meet their basic needs and expand their opportunities
+to reach their full potential_". (21) ~ https://www.unicef.org/about/who/index_mission.html
+
+![unicef-mission-statement](https://user-images.githubusercontent.com/194400/65377978-0e102180-dcaa-11e9-8bc3-7766251aa9a2.png)
+
+### charity:water
+
+"_charity:water is a nonprofit organization bringing clean and safe drinking water to people in developing countries._" (16) ~ https://my.charitywater.org/about/mission
+
+![charity:water-mission-statement](https://user-images.githubusercontent.com/194400/65378158-c0e17f00-dcac-11e9-9d27-ce4876a745e4.png)
+
+
+### Tesla
+
+"_Tesla’s mission is to accelerate the world’s transition to sustainable energy._"
+
+![tesla-mission-statement](https://user-images.githubusercontent.com/194400/65378132-752ed580-dcac-11e9-9cad-6d5619ad91f8.png)
+
+
+<hr />
+
+If you more find good examples of mission statements,
+please add them!
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat-square)](https://github.com/dwyl/start-here/issues)
+
+<hr />
+
+
+# What is _@dwyl's_ `Mission`?
+
+### "_Empower people to maximise their personal effectiveness, creativity and happiness._"
```

Comment:
❤️  I would be tempted to add 'whilst reducing their environmental impact' but I don't want to make the mission:
a) murky
b) appear to only appeal to the Western world/people of certain means (which traditionally caring for the environment has been)

We have at various points also discussed the possibility of adding 'through the use of technology' or words to that effect. I am motivated by a technology-driven organisation and that's where my heart lies but the mission will evolve as we ask questions of it in the short term.

Maybe we can revise this to add it in at a later date, definitely not needed for a starting point.

Reply:
@iteles indeed we have discussed the wording "_through technology_" in the past.
The _reason_ I've left it out (_this time_) is:
a) to keep it succinct 📏 
b) often doing what you love will involve _escaping_ from tech 📵 

We could even remove creativity and happiness and _just_ focus on ***personal effectiveness***,
but I feel that removing the "joy" from our mission would make us robots and nobody wants that. 😉